### DISCUSSION TITLE
xit for helix

### DISCUSSION THREAD

**Discussion by ChemicalXandco** [jotaen/xit#47] [Show & tell]
I have added syntax highlighting using [this](https://github.com/synaptiko/tree-sitter-xit) tree sitter grammar: https://github.com/helix-editor/helix/pull/3521. Now it is in the release version 22.08 so helix can be added to the website tooling section!

**Comment by jotaen** [comment]
Awesome, thanks for your work 🎉 Cool to see how the tree sitter grammar can be plugged into Helix. I’ve added a link to Helix to the [[x]it! website](https://xit.jotaen.net).
